src/mui/components/dialog/Dialog.tsx

```diff
@@ -107,7 +112,8 @@ function DialogModal({
                     color="neutral"
                     data-dismiss="modal"
                     aria-label={cancelButtonText}
-                    onClick={handleCancel}>
+                    onClick={handleCancel}
+                    {...cancelButtonProps}>
```

Comment:
Any way we can avoid spreading here and below?

Reply:
The idea was to allow passing any MUI button properties here so that in the future if any styles changed, we could pass new properties here without any modifications inside the cove.
For example, right now the DELETE button of the confirm dialog must be red:
```
<Dialog submitButtonProps={{color: "error"}} />
```
If we need to add an icon, we can just add the `startIcon` property, because we support all MUI Button properties here
```
<Dialog submitButtonProps={{color: "error", startIcon: <DeleteIcon />}} />
```

Reply:
Without spreading, this small change will require two PRs: one for web-app with passing the `startIcon` property, and one for cove with defining the `startIcon` property